tests/src/test/java/alluxio/client/rest/AlluxioMasterRestApiTest.java

```diff
@@ -110,11 +121,35 @@ public void getLostWorkers() throws Exception {
   }
 
   @Test
-  public void getMetrics() throws Exception {
+  public void getMetricsInfo() throws Exception {
     assertEquals(Long.valueOf(0),
         getInfo(NO_PARAMS).getMetrics().get(MetricsSystem.getMetricName("CompleteFileOps")));
   }
 
+  @Test
+  public void getUfsMetrics() throws Exception {
+    int len = 100;
+    FileSystemTestUtils.createByteFile(mResource.get().getClient(), "/f1", WritePType.THROUGH, len);
+    CommonUtils.waitFor("Metrics to be updated correctly", () -> {
+      try {
+        return FormatUtils.getSizeFromBytes(len).equals(getMetrics(NO_PARAMS)
+            .get("totalBytesWrittenUfs"));
```

Comment:
I tried to find a constant with this name somewhere in the codebase but didn't have any luck. It would probably be better than hard-coding the string in this test.

Reply:
Yeah its based on the variable name in `MasterWebUIMetrics` (we convert this to a json blob). Did you want a variable in the test or across the codebase, afaik we don't use the string form in other places.

Reply:
I think it would be nice if we used some kind of constant variable with the name across the codebase. But I think if we do that for the total "totalBytesWrittenUfs" we should apply it to other metrics in the JSON blob. I would say just don't worry about it.

I was thinking in MasterWebUIMetrics would have a set of constant variables representing the names in the JSON blob, so tha

```java
// in MasterWebUIMetrics.java
public static final String TOTAL_BYTES_WRITTEN_UFS = "totalBytesWrittenUfs";

// in AlluxioMasterRestApiTest.java
then in the test you could write
FormatUtils.getSizeFromBytes(len).equals(getMetrics(NO_PARAMS)
            .get(MasterWebUIMetrics.TOTAL_BYTES_WRITTEN_UFS));
```